pyharmonysearch/harmony_search.py

```diff
@@ -110,10 +113,14 @@ def run(self):
         # harmony_memory stores the best hms harmonies
         self._harmony_memory = list()
 
-        # fill harmony_memory using random parameter values
-        self._initialize()
+        # harmony_history stores all hms harmonies every generation (or n_inds generations)
```

Comment:
Instead of `n_inds`, do you mean to use `n`?

Reply:
Yes, n!